app/renderer/lib/client-frameworks/java.js

```diff
@@ -72,31 +63,26 @@ ${this.indent(code, 4)}
 `;
   }
 
-  codeFor_executeScript (/*varNameIgnore, varIndexIgnore, args*/) {
-    return `/* TODO implement executeScript */`;
-  }
-
   codeFor_findAndAssign (strategy, locator, localVar, isArray) {
     let suffixMap = {
-      xpath: 'XPath',
-      'accessibility id': 'AccessibilityId',
-      'id': 'Id',
-      'class name': 'ClassName',
-      'name': 'Name',
-      '-android uiautomator': 'AndroidUIAutomator',
-      '-android datamatcher': 'AndroidDataMatcher',
-      '-android viewtag': 'AndroidViewTag',
-      '-ios predicate string': 'IosNsPredicate',
-      '-ios class chain': 'IosClassChain',
+      'xpath': 'xpath',
+      'accessibility id': 'accessibilityId',
+      'id': 'id',
+      'class name': 'className',
+      'name': 'name',
+      '-android uiautomator': 'androidUIAutomator',
+      '-android datamatcher': 'androidDataMatcher',
+      '-android viewtag': 'androidViewTag',
+      '-ios predicate string': 'iOSNsPredicateString',
+      '-ios class chain': 'iOSClassChain',
     };
     if (!suffixMap[strategy]) {
-      throw new Error(`Strategy ${strategy} can't be code-gened`);
+      throw new Error(`Strategy '${strategy}' can't be code-gened`);
```

Comment:
We did not know how to generate the code for the '${strategy}' location strategy